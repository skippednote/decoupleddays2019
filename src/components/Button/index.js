import React from "react";

function Button({ children, className, clickHandler, ...rest }) {
  return (
    <button onClick={clickHandler} className={className} {...rest}>
      {children}
    </button>
  );
}

export default Button;
