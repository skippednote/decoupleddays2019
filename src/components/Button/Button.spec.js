import React from "react";
import { render } from "@testing-library/react";
import Button from ".";

it("renders the button component", () => {
  const { container } = render(<Button className={"button"}>New York</Button>);

  expect(container).toMatchSnapshot();
});
