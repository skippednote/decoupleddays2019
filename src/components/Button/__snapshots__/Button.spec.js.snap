// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the button component 1`] = `
<div>
  <button
    class="button"
  >
    New York
  </button>
</div>
`;
